src/view/com/auth/SplashScreen.web.tsx

```diff
@@ -90,6 +90,7 @@ const styles = StyleSheet.create({
   containerInner: {
     height: '100%',
     justifyContent: 'center',
+    // @ts-ignore web only
```

Comment:
Almost all of these are due to new types trying to protect us from ourselves: https://github.com/facebook/react-native/issues/39179